lib/mix/phoenix.ex

```diff
@@ -80,8 +80,8 @@ defmodule Mix.Phoenix do
       {k, {:array, _}} -> {k, []}
       {k, :belongs_to} -> {k, nil}
       {k, :integer}    -> {k, 42}
-      {k, :float}      -> {k, "120.5"}
-      {k, :decimal}    -> {k, "120.5"}
+      {k, :float}      -> {k, 120.5}
+      {k, :decimal}    -> {k, Decimal.new(43.20)}
```

Comment:
I only see `params` passed to the `model_test` and `controller_test`s for html and json. They don't seem to affect changesets or the form.

Anyway, no error, but saw the sample data generated in the tests used the string "120.5" for decimal. Ran  `validate_number/3` for the decimal field and it would always pass with the string. Checked out the docs in `Schema` and saw that `:decimal` uses a `Decimal` module. Using the `Decimal` module makes the validation work as expected.

So thought of passing `Decimal` and a `float`. The `Decimal.new` above makes 2 tests fail though and will need to change. Should I continue to change or do we close this? :)